core/src/main/resources/nitf_spec.xml

```diff
@@ -199,6 +199,214 @@
+        <field name="CG_INCIDENT" length="7" type="real" minval="0.0" maxval="89.9999" unit="degrees"/>
+        <field name="CG_SLOPE" length="7" type="real" minval="-89.9999" maxval="89.9999" unit="degrees"/>
+        <field name="CG_TILT" length="8" type="real" minval="-44.9999" maxval="44.9999" unit="degrees"/>
+        <field name="CG_LD" length="1" type="string"/>
+        <field name="CG_NORTH" length="8" type="real" minval="0.0" maxval="359.9999" unit="degrees"/>
+        <field name="CG_NORTH_CONFIDENCE" length="6" type="real" minval="0.0" maxval="9.9999" unit="degrees"/>
+        <field name="CG_EAST" length="8" type="real" minval="0.0" maxval="359.9999" unit="degrees"/>
+        <field name="CG_RLOS" length="8" type="real" minval="0.0" maxval="359.9999" unit="degrees"/>
+        <field name="CG_LOS_CONFIDENCE" length="6" type="real" minval="0.0" maxval="9.9999" unit="degrees"/>
+        <field name="CG_LAYOVER" length="8" type="real" minval="0.0" maxval="359.9999" unit="degrees"/>
+        <field name="CG_SHADOW" length="8" type="real" minval="0.0" maxval="359.9999" unit="degrees"/>
+        <field name="CG_OPM" length="7" type="real" minval="0.0" maxval="999.999" unit="miliarcsec"/>
+        <field name="CG_MODEL" length="5" type="string"/>
+        <field name="CG_AMPT_X" length="13" type="real" minval="-99999999.999" maxval="99999999.999"/>
+        <field name="CG_AMPT_Y" length="13" type="real" minval="-99999999.999" maxval="99999999.999"/>
+        <field name="CG_AMPT_Z" length="13" type="real" minval="-99999999.999" maxval="99999999.999"/>
+        <field name="CG_AP_CONF_XY" length="6" type="real" minval="0.0" maxval="999.99" unit="meters"/>
+        <field name="CG_AP_CONF_Z" length="6" type="real" minval="0.0" maxval="999.99" unit="meters"/>
+        <field name="CG_APCEN_X" length="13" type="real" minval="-99999999.999" maxval="99999999.99"/>
+        <field name="CG_APCEN_Y" length="13" type="real" minval="-99999999.999" maxval="99999999.99"/>
+        <field name="CG_APCEN_Z" length="13" type="real" minval="-99999999.999" maxval="99999999.99"/>
+        <field name="CG_APER_CONF_XY" length="6" type="real" minval="0.0" maxval="999.99" unit="meters"/>
+        <field name="CG_APER_CONF_Z" length="6" type="real" minval="0.0" maxval="999.99" unit="meters"/>
+        <field name="CG_FPNUV_X" length="9" type="real" minval="-1.0" maxval="1.0"/>
+        <field name="CG_FPNUV_Y" length="9" type="real" minval="-1.0" maxval="1.0"/>
+        <field name="CG_FPNUV_Z" length="9" type="real" minval="-1.0" maxval="1.0"/>
+        <field name="CG_IDPNUVX" length="9" type="real" minval="-1.0" maxval="1.0"/>
+        <field name="CG_IDPNUVY" length="9" type="real" minval="-1.0" maxval="1.0"/>
+        <field name="CG_IDPNUVZ" length="9" type="real" minval="-1.0" maxval="1.0"/>
+        <field name="CG_SCECN_X" length="13" type="real" minval="-99999999.999" maxval="99999999.999" unit="meters"/>
```

Comment:
The units can be degrees or metres here (depends on `CG_MODEL` value). Suggest just removing it for the X and Y cases.